doc/classes/ScriptLanguageExtension.xml

```diff
@@ -358,16 +358,12 @@
 		<constant name="LOOKUP_RESULT_MAX" value="9" enum="LookupResultType">
 		</constant>
 		<constant name="LOCATION_LOCAL" value="0" enum="CodeCompletionLocation">
-			The option is local to the location of the code completion query - e.g. a local variable.
 		</constant>
 		<constant name="LOCATION_PARENT_MASK" value="256" enum="CodeCompletionLocation">
-			The option is from the containing class or a parent class, relative to the location of the code completion query. Perform a bitwise OR with the class depth (e.g. 0 for the local class, 1 for the parent, 2 for the grandparent, etc) to store the depth of an option in the class or a parent class.
 		</constant>
 		<constant name="LOCATION_OTHER_USER_CODE" value="512" enum="CodeCompletionLocation">
-			The option is from user code which is not local and not in a derived class (e.g. Autoload Singletons).
 		</constant>
 		<constant name="LOCATION_OTHER" value="1024" enum="CodeCompletionLocation">
-			The option is from other engine code, not covered by the other enum constants - e.g. built-in classes.
```

Comment:
This were like lost by mistake?

Reply:
This was removed and put in `code_edit.xml` it felt better as this was what was done for the other shared enum

Reply:
Well this class still needs documentation too, so we shouldn't remove docs. It's not uncommon to have duplicate descriptions for duplicated APIs.

Reply:
Thought this was intended as this class does not have any documentation. Then should I also duplicate the doc for the other enum that is shared?